lib/internal/cluster/master.js

```diff
@@ -12,6 +12,16 @@ const cluster = new EventEmitter();
 const intercom = new EventEmitter();
 const SCHED_NONE = 1;
 const SCHED_RR = 2;
+var debugPortOffset = 0;
+
+const debugArgRegex = /^--(inspect|debug(?:-brk|port)?)(?:=.*)?$/;
```

Comment:
Nope: it can be host:address, ipv4 or ipv6 address. That's why i've came with more complex parsing solution